pages/tabs/actions.page.tsx

```diff
@@ -54,86 +56,90 @@ export default function TabsDemoPage() {
         'Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.',
     },
   ];
-  const [tabsXORActions, setTabsXORActions] = useState([
-    {
-      label: 'First tab',
-      id: 'first',
-      dismissible: true,
-      dismissLabel: 'Dismiss first tab (xor actions variant)',
-      onDismiss: () => setTabsXORActions(prevTabs => prevTabs.slice(1)),
-      content: (
-        <>
-          Diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et
-          accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum
-          dolor sit ameta.,
-        </>
-      ),
-    },
-    {
-      label: 'Second tab',
-      id: 'second',
-      action: <Button iconName="copy" variant="icon" ariaLabel="Action button for the Second tab" />,
-      content:
-        'Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.',
-    },
-  ]);
+  const [tabsXORActions, setTabsXORActions] = useState(
+    (): Array<TabsProps.Tab> => [
```

Comment:
Fixing type issue here and in all other tab pages. `onDismiss` handlers call `setTabsXORActions` which creates circular type, which typescript does not like